lib/wallet/rpc.js

```diff
@@ -352,6 +354,26 @@ class RPC extends RPCBase {
     return null;
   }
 
+  async rescan(args, help) {
+    if (help || args.length > 1)
+      throw new RPCError(errs.MISC_ERROR, 'rescan "height"');
```

Comment:
optional params need ()

```suggestion
      throw new RPCError(errs.MISC_ERROR, 'rescan ( "height" )');
```